CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ if (FL_BUILD_TESTS)
   include(TestUtils)
 endif ()
 
+find_package(OpenMP) # Needed for linking with OpenMP_CXX by the audio ext
```

Comment:
@jacobkahn do we need this?

Reply:
I do need it to avoid some linking failures when building without MKL.

Reply:
cc @jacobkahn to sync on this

Reply:
now a separate PR:
https://github.com/facebookresearch/flashlight/pull/285